compiler/z/codegen/OMRCodeGenerator_inlines.hpp

```diff
@@ -0,0 +1,31 @@
+/*******************************************************************************
+ *
+ * (c) Copyright IBM Corp. 2000, 2017
+ *
+ *  This program and the accompanying materials are made available
+ *  under the terms of the Eclipse Public License v1.0 and
+ *  Apache License v2.0 which accompanies this distribution.
+ *
+ *      The Eclipse Public License is available at
+ *      http://www.eclipse.org/legal/epl-v10.html
+ *
+ *      The Apache License v2.0 is available at
+ *      http://www.opensource.org/licenses/apache2.0.php
+ *
+ * Contributors:
+ *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ ******************************************************************************/
+
+#ifndef OMRCODEGENERATOR_INLINES_INCL
```

Comment:
This #ifdef is too similar to it's parent in OMR, and offers no indication that its for the Z codegen.  Please use `OMR_Z_CODEGENERATOR_INLINE_INCL` instead.